Riot/Modules/Room/TimelineCells/Common/MXKRoomBubbleTableViewCell.m

```diff
@@ -237,6 +237,7 @@ - (void)setupMessageTextView
     [tapGesture setDelegate:self];
     [self.messageTextView addGestureRecognizer:tapGesture];
     self.messageTextView.userInteractionEnabled = YES;
+    self.messageTextView.clipsToBounds = NO;
```

Comment:
Is this really needed?

Reply:
When you long press on a link iOS creates a box with a shadow around it before it opens the preview.
Without this line that box is clipped off by the UITextView itself. :-/ 
Does it make sense?